subcommands/keys/tuf_updates_review.go

```diff
@@ -101,20 +103,50 @@ func doTufUpdatesReview(cmd *cobra.Command, args []string) {
 				}
 			}
 		}
+	} else if updates.Status == client.TufRootUpdatesStatusNone {
+		fmt.Println("There are no TUF root updates in progress.")
+		if len(updates.Advices.Mandatory) > 0 {
+			fmt.Println(`\nFoundries.io identified some problems with your existing TUF metadata.
```

Comment:
We should probably avoid the topic of branding and just say. "Problems have been identified with your TUF metadata"

Reply:
Makes sense

Reply:
We now have no errors returned for this use case from the API (as per server-side review thread).

So this code was removed. I applied the branding part elsewhere.